### ISSUE TITLE
Mana cost - Suggestion

### ISSUE THREAD

**Issue by Moonlight-Cockatrice** [Cockatrice/Cockatrice#3655]
<b>System Information:</b>
  

  
__________________________________________________________________________________________

  
Rearrange description window to show Mana cost where 'Color identity' currently is. Currently it's near the bottom which is the opposite of where eyes are usually drawn when looking for mana costs to play cards. 

The description window is very busy right now, given it displays converted mana cost, mana cost, color, _and_ color identity. The ulterior most being only relevant in EDH and only when choosing a commander while unfamiliar with rules on color identity. It honestly doesn't seem necessary.

**Comment by ctrlaltca** [comment]
Entries in the card info are dynamic, so I'm not sure what you are talking about is the same as i see:
![Schermata 2019-03-11 alle 08 41 20](https://user-images.githubusercontent.com/1631111/54107511-ae22ef00-43d9-11e9-8e91-1c60bdb9cb0e.png)

**Comment by Moonlight-Cockatrice** [comment]
That's interesting. I see this:

![](https://i.gyazo.com/0b955e7614975477930dcb01986b5e57.png)

**Comment by Moonlight-Cockatrice** [comment]
I'm not sure dynamic card descriptions are a good thing

**Comment by ctrlaltca** [comment]
We have cards with a list of name => value properties now.
We could hardcode a specific order and show/hidden for each property name, but other problems would arise:
 * not all games use the same property names, so maybe we hide important properties for other games
 * every user would like a different order, or to hide that specific property they're not interested to, etc..

What's a possible solution? Maybe an user preference panel where you can decide what properties youw ant to show/hide, and the order they should appear?

**Comment by Moonlight-Cockatrice** [comment]
I think adding preferences is a great idea. Something under View that has checkmarks for what you do and don't want to see in the description window.

**Comment by MotherofRuins** [comment]
The same properties should always show up on the same line.  Leave blank lines if a card does not have that property.  Top lines should be Name, Mana Cost, Color(s), Card Type, P/T.  While different users will have different preferences, I'm pretty sure those are the most important for all.  Side, Layout, Main Card Type, Converted Mana Cost, Color Identity less important (to me).  

When listing colors, use the same canonical order that Wizards uses (WUBRG?).  Eg, Aeromonculus has Color Identity GU, Mana Cost 1GU, and Colors UG.  

The property values show up with vertically aligned with different "tab stops" even when all the properties are the same.  

Basically, when I am running down through a list in deck editor using down arrow, or moving the mouse over multiple cards in game or game lobby, as it changes from card to card, the rows and columns shouldn't jump around.  For property values that want to wrap text, I would keep on one line, and truncate with "..." at the end to user know they need a wider Card Info element.

**Comment by ebbit1q** [comment]
I think users should be able to manually customise this through settings and have it store them, this'd require some ui overhaul but nothing too major.

**Comment by tooomm** [comment]
As used to from other applications, a right click and edit/customise option or a little pencil icon in the corner to open a pop up to re-arrange and disable/enable certain information would be nice.

I think I mentioned something similar for the deck editor columns once, too...
